kite-morphlines/kite-morphlines-phonenumber/pom.xml

```diff
@@ -0,0 +1,69 @@
+<!--
+
+    Copyright (C) 2014 Stratio (http://stratio.com)
```

Comment:
License is fine, but copyright in all files should be Cloudera per https://github.com/kite-sdk/kite/wiki/How-to-contribute